controlplane/internal/controller/clusterdeployment_controller.go

```diff
@@ -327,5 +328,14 @@ func (r *ClusterDeploymentReconciler) computeAgentClusterInstall(
 			ManifestsConfigMapRefs: additionalManifests,
 		},
 	}
+
+	if len(acp.Spec.AgentConfigSpec.APIVIPs) > 0 && len(acp.Spec.AgentConfigSpec.IngressVIPs) > 0 {
+		aci.Spec.APIVIPs = acp.Spec.AgentConfigSpec.APIVIPs
+		aci.Spec.IngressVIPs = acp.Spec.AgentConfigSpec.IngressVIPs
```

Comment:
What version of assisted-service will our operator support?

I ask because previous version require `aci.Spec.APIVIP` and I'm wondering if we should have that too?

Reply:
I think this was quite some versions ago, but we can check and make sure, and either:
* before such version, just set APIVIP instead (maybe just pick the first item to keep our API simpler)
* Do not allow installs of what it's probably ancient versions
WDYT?

Reply:
The second option is more favorable imo, since we don't want to extend support to prior versions. Do you think it'll be difficult to detect which version is deployed?

Reply:
I have added naive checking vs the string version, I think we could introspect the image and extract `io.openshift.release` or something to make it better, I will create an issue for that.

Reply:
Nice! Yeah sounds good